### DISCUSSION TITLE
Import into table that has additional columns

### DISCUSSION THREAD

**Discussion by SuperBrain** [dbeaver/dbeaver#18730] [Q&A]
I need to import data from my old database into slightly restructured tables. Some destination tables now have additional columns, but the Import dialog lists only matching columns.

Is there a way, using either Import or Export dialog, to have DBeaver list all destination columns and allow me to specify default values for unmatched ones?

**Comment by LonwoLonwo** [comment]
> Is there a way, using either Import or Export dialog, to have DBeaver list all destination columns and allow me to specify default values for unmatched ones?

No, unfortunately, DBeaver doesn't have such an option. You need to specify the default column status manually and then use import.

**Comment by LonwoLonwo** [comment]
Or! You can use column transformer during the table transfer.
You can choose constant values for the new column, NULL or expression 

![2023-02-22 20_34_59-Data Transfer](https://user-images.githubusercontent.com/45152336/220710038-50b11f42-baf1-4a8a-947e-d25fecce8f69.png)

![2023-02-22 20_35_28-Data Transfer](https://user-images.githubusercontent.com/45152336/220710044-527e5901-3076-499a-b4d2-34dd8004ea33.png)
